src/views/Home.jsx

```diff
@@ -1,16 +1,23 @@
 import './Home.css';
-
-export function Home({ data, setListPath }) {
+import { SingleList } from '../components';
+export function Home({ data, setListPath, path }) {
 	return (
 		<div className="Home">
 			<p>
 				Hello from the home (<code>/</code>) page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `lists` array
-				 * so we can see which lists the user has access to.
-				 */}
+				{data &&
+					data.map((list) => {
+						return (
+							<SingleList
+								key={crypto.randomUUID()}
+								name={list.name}
+								setListPath={setListPath}
+								path={path}
```

Comment:
This path actually comes from the list item itself!
```suggestion
								path={list.path}
```